server/index.js

```diff
@@ -119,4 +119,13 @@ app.get("/v1/query", async (req, res) => {
   res.send(await stores.findAllStores());
 });
 
+app.get("/v2/query", async (req, res) => {
+  let query = {
+    lat: req.query.lat,
+    lng: req.query.lng,
+    radius: req.query.radius,
+    page: req.query.page}
```

Comment:
page doesn't seem to be used?

Reply:
It is for future use cases. We can order the result by the distance and use pagination inbuilt into backend

Reply:
sounds good